WordPressKit/WordPressComServiceRemote.h

```diff
@@ -48,6 +48,25 @@ typedef void(^WordPressComServiceFailureBlock)(NSError *error);
                              success:(WordPressComServiceSuccessBlock)success
                              failure:(WordPressComServiceFailureBlock)failure;
 
+/**
+ * @brief Create a new WordPress.com account from Apple ID credentials.
+ *
+ * @param token          Token provided by Apple.
+ * @param email          Apple email to use for new account.
+ * @param userName       The username for the new account. Formed from the Apple ID fullname.
```

Comment:
I think this is the user's name, not their username?

Reply:
Gah. Clearly I had the endpoint param name on my brain. I'll change it. Thanks.